md5/src/compress/loongarch64_asm.rs

```diff
@@ -0,0 +1,206 @@
+//! LoongArch64 assembly backend
+
+use core::arch::asm;
+
+const RC: [u32; 64] = [
```

Comment:
This constant is quite big, so it's better to create `src/consts.rs` and move it there. Use it both here and in the software backend. You can add `#[allow(dead_code)]` for it to work around warning which will be emitted with enabled `asm` feature.